01week/rockPaperScissors.js

```diff
@@ -12,6 +12,62 @@ function rockPaperScissors(hand1, hand2) {
 
   // Write code here
 
+  //First will convert the input to lowercase. Then will check if the input is correct ( rock, paper, scissors). 
+  // If the input is not correct the program won't run and it'll show "Wrong input! Please enter rock,paper or scissors."
+  //Next will compare the two inputs and show the winner.
+  
+  hand1= hand1.toLowerCase().trim();
+  hand2= hand2.toLowerCase().trim();
+
+  if((hand1=='rock' || hand1=='paper' || hand1== 'scissors') && (hand2=='rock' || hand2=='paper' || hand2=='scissors'))
```

Comment:
You are repeating yourself a lot in if/else statement, find a way to simplify. Also brackets should be in the following arrangement for readability. Please fix all of the if/else statments
```javascript
if (hand1 == 'rock'){
    return ... 
}
```